docs/src/tutorials/applications/optimal_power_flow.jl

```diff
@@ -23,13 +21,15 @@
 # [Example: quantum state discrimination](@ref) tutorial, and see the
 # [Complex number support](@ref) section of the manual for more details.
 
-# This tutorial takes a matrix-oriented approach focused on network nodes
-# that simplifies the construction of semidefinite programs.
-# Another approach is to formulate the problem focusing on network lines
-# where it is easier to work with flow constraints. A general approach is
-# provided by the Julia/JuMP package [PowerModels.jl](https://lanl-ansi.github.io/PowerModels.jl/stable/),
-# an open-source framework to a broad range of power flow model formulations
-# along with utilities for working with detailed network data.
+# !!! info
+#     This tutorial takes a matrix-oriented approach focused on network nodes
+#     that simplifies the construction of semidefinite programs.
+#     Another approach is to formulate the problem focusing on network lines
+#     (known as a _branch model_) where it is easier to work with flow 
+#     constraints. A general approach is provided by the Julia/JuMP package 
```

Comment:
```suggestion
#     constraints. A general approach is provided by
```

Reply:
I don't know who started the Julia/JuMP thing. I dislike it, but it seems pretty popular. Since we're in the JuMP docs though, readers should be familiar with Package.jl for a Julia package.